views/admin/admin-page.php

```diff
@@ -47,7 +48,7 @@
 				<h2 title="Get the most out of Jetpack with these features"><?php _e( 'Get the most out of Jetpack with...', 'jetpack' ); ?></h2>
 				<div class="modules"></div>
 				<?php if ( current_user_can( 'jetpack_manage_modules' ) ) : ?>
-				<a href="<?php echo admin_url( 'admin.php?page=jetpack_modules' ); ?>" class="button" ><?php esc_html_e( 'See the other 25 Jetpack features', 'jetpack' ); ?></a>
+				<a href="<?php echo admin_url( 'admin.php?page=jetpack_modules' ); ?>" class="button" ><?php echo sprintf( __('See the other %s Jetpack features', 'jetpack' ), count( Jetpack::get_available_modules() ) - count( $data['recommended_list'] ) ); ?></a>
```

Comment:
@dereksmart, can you make a second PR with just this change in it, so we can tag it in for 3.4.1?  This is an important fix that we don't want to lose track of